### PR TITLE
Sboeing dec 2021

### PR DESCRIPTION
**Pull request by sjboeing**
Changing the naming of L_c to L_v, and adding a python script to convert to netcdf

### CODE REVIEW

**Comment by matt-frey** [comment]
@sjboeing I changed the base branch from `main` to `3d-model`.

File: h5_to_nc.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import os.path
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('input_file_name',type=str)
+args = parser.parse_args()
+input_file_name=args.input_file_name
+
+file_ext = os.path.splitext(input_file_name)[1]
+file_root = os.path.splitext(input_file_name)[0]
```
Comment (matt-frey) [inline h5_to_nc.py:12]: This could be simplified to
```Python
file_root, file_ext = os.path.splitext(intput_file_name)
```

**Review by matt-frey — APPROVED** [review]
Hi Steef, you can pull the PR. I have just a single comment.

File: h5_to_nc.py (hunk 1 above)
Comment (sjboeing) [reply h5_to_nc.py]: Thanks, good plan!

**Pull request merged**